reinforcement_learning/Lesson-3-Value-Based-Learning-Method-draft.ipynb

```diff
@@ -0,0 +1,1158 @@
+{
```

Comment:
<ul><li>Is this a typo: "There is on 1 optimal path (fewest blocks to take)"? I'm guessing it's supposed to say, "There is only 1 optimal path (fewest blocks to take)"</li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BreakoutMentors/Data-Science-and-Machine-Learning/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>